### PR TITLE
game play API Integration

### PR DESCRIPTION
**Pull request by Janvi-kalavadiya-scaletech**
## Changes

- Dice play API Integration
- Wallet Amount API Integration
- Access token regenerate 
- Design changes

### CODE REVIEW

File: src/assets/styles/common/common.scss [hunk 1]
```diff
@@ -1364,4 +1364,10 @@
 
 .no-pointer-events {
     pointer-events: none;
-}
\ No newline at end of file
+}
+.modal-dialog {
+    position: relative;
+    margin-left: auto;
```
Comment (jasmeen-goyani-scaletech) [inline src/assets/styles/common/common.scss]: use margin: 0 auto

File: src/features/dashboard/container/dashboardContainer.tsx [hunk 2]
```diff
@@ -1,129 +1,187 @@
...
+                getPlayData();
+            })
+            .catch((err: Error) => {
+                dispatch(createAction(actionTypes.AUTH_FAILED));
+                console.error('Error', err);
+            });
+    };
+
+    const logOut = () => {
+        dispatch(createAction(actionTypes.AUTH_LOGOUT));
```
Comment (jasmeen-goyani-scaletech) [inline src/features/dashboard/container/dashboardContainer.tsx:134]: Add this in reducer file

File: src/features/dashboard/container/dashboardContainer.tsx [hunk 3]
```diff
@@ -136,21 +194,24 @@ const Dashboard = () => {
                                 ))}
                             </div>
 
-                            <div className="dice-side-wrapper flex  mt--50">
-                                {staticDice.map((data, index) => (
+                            <div
+                                className={`dice-side-wrapper flex  mt--50  ${
+                                    isPlay && 'disabled no-pointer-events'
+                                }`}>
+                                {staticDice.map((data, index: number) => (
```
Comment (jasmeen-goyani-scaletech) [inline src/features/dashboard/container/dashboardContainer.tsx]: no need to write type for index

File: src/features/dashboard/container/dashboardContainer.tsx [hunk 4]
```diff
@@ -187,41 +248,65 @@ const Dashboard = () => {
...
-                            className="play-btn border-radius--half custom-btn cursor--pointer font-size--25 text--white position--relative overflow--hidden"
+                            className='play-btn border-radius--half custom-btn cursor--pointer font-size--25 text--white position--relative overflow--hidden'
                             disabled={guessVal <= 0}
-                            onClick={handleDiceClick}>
+                            onClick={getPlayData}>
                             {!isPlay && (
                                 <i
-                                    className="fa fa-play play flex justify-content--center align-items--center"
-                                    id="play"></i>
+                                    className='fa fa-play play flex justify-content--center align-items--center'
```
Comment (jasmeen-goyani-scaletech) [inline src/features/dashboard/container/dashboardContainer.tsx]: Write down **i** tag in optimize way

**Review by jasmeen-goyani-scaletech — CHANGES_REQUESTED** [review]

File: src/assets/styles/common/common.scss (hunk 1 above)
Comment (Janvi-kalavadiya-scaletech) [reply src/assets/styles/common/common.scss]: done

File: src/features/dashboard/container/dashboardContainer.tsx (hunk 2 above)
Comment (Janvi-kalavadiya-scaletech) [reply src/features/dashboard/container/dashboardContainer.tsx:134]: done

File: src/features/dashboard/container/dashboardContainer.tsx (hunk 3 above)
Comment (Janvi-kalavadiya-scaletech) [reply src/features/dashboard/container/dashboardContainer.tsx]: Done

File: src/features/dashboard/container/dashboardContainer.tsx (hunk 4 above)
Comment (Janvi-kalavadiya-scaletech) [reply src/features/dashboard/container/dashboardContainer.tsx]: done

**Review by jasmeen-goyani-scaletech — APPROVED** [review]

**Pull request merged**